src/apps/bridge/aanderaaController.cpp

```diff
@@ -298,11 +272,10 @@ static void createAanderaaSub(uint64_t node_id) {
 
   new_sub->node_id = node_id;
   new_sub->next = NULL;
-  new_sub->current_agg_period_ms = (DEFAULT_CURRENT_AGG_PERIOD_MIN * 60 * 1000);
-  uint32_t agg_period_min;
-  if (_ctx._usr_cfg->getConfig("currentAggPeriodMin", strlen("currentAggPeriodMin"),
-                               agg_period_min)) {
-    new_sub->current_agg_period_ms = (agg_period_min * 60 * 1000);
```

Comment:
this config "currentAggPeriodMin" controlled when the aanderaa controller would compute the mean/stdev. This isn't connected at all to the sample duration and thus could be completely different. (i.e. multiple aggregations per sample duration, or multiple sample durations per aggregation. From "The Bridge shouldn’t have its own currentAggPeriodMin setting - it should just be using the sample duration / sample interval to drive aggregation timing. For now and foreseeable future wavefleet and other consumers of the data will be expecting 1::1 aggregation::sample period." this config needed to be removed and instead the mean/ stdev calculation trigger needs to be at the end of a sample duration.